bin/validate-json

```diff
@@ -198,21 +210,21 @@ try {
     $urlSchema = $resolver->resolve($pathSchema, $urlData);
 
     if (isset($arOptions['--dump-schema-url'])) {
```

Comment:
Not sure whether you're intending this one in a different commit to the `--dump-schema` one, or if you missed it - but in the past you've combined this kind of thing in a single commit, so adding a comment just in case.